pkg/kubelet/kubelet.go

```diff
@@ -81,6 +84,7 @@ type SyncHandler interface {
 	// Syncs current state to match the specified pods. SyncPodType specified what
 	// type of sync is occuring per pod. StartTime specifies the time at which
 	// syncing began (for use in monitoring).
+	SyncDockerPods(pods []api.BoundPod, podSyncTypes map[types.UID]metrics.SyncPodType, startTime time.Time) error
```

Comment:
Well, that's easy, you just add a kubelet argument to the function. You can leave it like this for now if you want, I just think e.g. the if statement on 1572 is icky and I wouldn't be surprised if they don't like it... but who knows :-)